src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * File containing the ContentViewController class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    protected $defaultSortFields = [
+        Location::SORT_FIELD_NAME => ['key' => 'sort.name', 'default' => 'Content name'],
+        Location::SORT_FIELD_PRIORITY => ['key' => 'sort.priority', 'default' => 'Priority'],
+        Location::SORT_FIELD_MODIFIED => ['key' => 'sort.modified', 'default' => 'Modification date'],
+        Location::SORT_FIELD_PUBLISHED => ['key' => 'sort.published', 'default' => 'Publication date']
+    ];
+
+    protected $otherSortFields = [
+        Location::SORT_FIELD_PATH => ['key' => 'sort.path', 'default' => 'Location path'],
+        Location::SORT_FIELD_CLASS_IDENTIFIER => [
+            'key' => 'sort.content.type.identifier',
+            'default' => 'Content type identifier'
+        ],
+        Location::SORT_FIELD_SECTION => ['key' => 'sort.section', 'default' => 'Section'],
+        Location::SORT_FIELD_DEPTH => ['key' => 'sort.depth', 'default' => 'Location depth'],
+        Location::SORT_FIELD_CLASS_NAME => ['key' => 'sort.content.type.name', 'default' => 'Content type name']
+    ];
+
+    public function detailsTabAction(ContentView $view)
+    {
+        $versionInfo = $view->getContent()->getVersionInfo();
+        $contentInfo = $versionInfo->getContentInfo();
+
+        $sectionService = $this->getRepository()->getSectionService();
+        $section = $sectionService->loadSection($contentInfo->sectionId);
+
+        $view->addParameters([
+            'section' => $section,
+            'contentInfo' => $contentInfo,
+            'versionInfo' => $versionInfo,
+            'creator' => $this->loadUser($contentInfo->ownerId),
+            'lastContributor' => $this->loadUser($versionInfo->creatorId),
+            'translations' => $this->getTranslations($versionInfo),
+            'ordering' => [
+                'sortFields' => $this->getSortFields($view->getLocation()->sortField),
+                'sortOrders' => $this->getSortOrders()
+            ]
+        ]);
+
+        return $view;
+    }
+
+    protected function loadUser($userId)
+    {
+        $userService = $this->getRepository()->getUserService();
+
+        return $this->getRepository()->sudo(function () use ($userId, $userService) {
```

Comment:
a bit late, but afaik you don't need to use sudo when using UserService, most load methods there don't check permissions on purpose for use cases where you need to load users _(REST however does check permissions when looking up users)_